apps/src/sites/studio/pages/lessons/show.js

```diff
@@ -2,16 +2,39 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import getScriptData from '@cdo/apps/util/getScriptData';
 import LessonOverview from '@cdo/apps/templates/lessonOverview/LessonOverview';
-import {sampleActivities} from '../../../../../test/unit/lib/levelbuilder/lesson-editor/activitiesTestData';
 import announcementsReducer, {
   addAnnouncement
 } from '@cdo/apps/code-studio/announcementsRedux';
 import {getStore} from '@cdo/apps/code-studio/redux';
 import {registerReducers} from '@cdo/apps/redux';
 import {Provider} from 'react-redux';
+import _ from 'lodash';
 
 $(document).ready(function() {
   const lessonData = getScriptData('lesson');
+  const activities = lessonData['activities'];
+
+  // Rename any keys that are different on the backend.
+  activities.forEach(activity => {
```

Comment:
we shouldn't bother with that because everything here should go away in both places once we change the client names to match what's used on the server. initializing missing arrays should be replaced by fixing the data that gets returned by the server too. once we've chipped away at it if it looks like there is any shareable code remaining then we could extract it then but in general I would expect most/all of this to go away.